src/static/js/pad.js

```diff
@@ -146,6 +146,13 @@ const getParameters = [
       Cookies.set('language', val);
     },
   },
+  {
+    name: 'focusOnEditor',
+    checkVal: 'false',
+    callback: (val) => {
+      settings.focusOnEditor = val;
+    },
```

Comment:
This is broken: it sets `settings.focusOnEditor` to the string `'false'` which is truthy. Here's a fix:

```suggestion
    callback: (val) => settings.focusOnEditor = val !== 'false',
```